src/showcase/Showcase.unit.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { render, screen } from '@testing-library/react'
+
+import { Paragraph } from '../paragraph'
+import { Showcase } from './index'
+
+describe('Showcase', () => {
+  it('Should render', () => {
+    render(
+      <Showcase>
+        <div>fhtagn</div>
+        <div>cthulhu</div>
+        <Paragraph>Paragraph</Paragraph>
+      </Showcase>,
+    )
+    expect(screen.getByRole('list')).toBeInTheDocument()
+    expect(screen.getAllByRole('listitem')).toHaveLength(3)
```

Comment:
Did you test to improve the test by checking the visibility of elements over time?

You can try a combination of `waitFor()` and `toBeVisible()`

Reply:
I tried your suggestion but couldn't make it more so far...

Reply:
As seen together, after multiple attempts, I added a TODO